components/Nav/Nav.test.tsx

```diff
@@ -64,13 +77,37 @@ describe('Nav', () => {
 
     const title = screen.getByTestId('title');
     const logo = screen.getByTestId('logo-nav');
+    const preferencesLink = screen.getByTestId('settings-link');
     const signOutButton = screen.getByTestId('sign-out-button');
 
+    expect(preferencesLink).toBeInTheDocument();
     expect(title).toBeInTheDocument();
     expect(logo).toBeInTheDocument();
     expect(signOutButton).toBeInTheDocument();
   });
 
+  it('should route to the preferences page when the user clicks on the preferences link', async () => {
+    mockUsePathname.mockReturnValue('/league/all');
+
+    render(<Nav />);
+
+    const drawerTrigger = screen.getByTestId('drawer-trigger');
+    fireEvent.click(drawerTrigger);
+
+    const preferencesLink = await screen.findByTestId('settings-link');
```

Comment:
We could hardcode `await screen.findByTestId('settings-link')` into lines 99 and 102 for cleaner code since we're only using it twice.

Reply:
Could do that but is it better to have them separate for legibility or condensed code? 🤔

Reply:
@choir27 @ryandotfurrer I dunno what to do now lol. Is it really cleaner? I like it for legibility. One could say legibility is clean code?

Reply:
I agree legibility is clean code; if you can't easy read the code, or it's complexity makes it hard to understand, it's not clean IMO.